package.json

```diff
@@ -13,7 +13,7 @@
     "smoketest": "npm run type-check && npm run lint && npm run format:check && npm run build",
     "type-check": "tsc --noEmit",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
-    "lint:fix": "npm run lint --fix",
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
I've seen this double `--` thing multiple times now, and I'm still not sure what it's good for... 😄 

I'm going to assume you tested that this works properly. 😉

Reply:
The double `--` is a bit difficult to explain, but I'll do my best:

With the previous command `npm run lint --fix` we were passing the fix option to the npm script itself. But what we want to do is to pass the fix option to the command executed under the hood, ESLint in this case. To do so, you need to indicate that with a `--` before whatever you want to pass to such command. Now, we are passing the fix option properly to ESLint with this trick!

Reply:
Ah that's a lot clearer now! Thank you! 😄